frontend/src/share/SearchPanel/SearchPanel.scss

```diff
@@ -1,15 +1,31 @@
-@import '../../App';
+@import '../../styles/colors.scss';
 
 .search-group {
     min-height: 85px;
-
-    @media (max-width: 500px) {
-        width: 85% !important;
+    padding: 20px 20px 30px;
+    border-radius: 2px;
+    margin: 10px;
+    border: 1px solid $light-gray;
+    -webkit-box-shadow: 4px 4px 8px -4px rgba(34, 60, 80, 0.2);
+    -moz-box-shadow: 4px 4px 8px -4px rgba(34, 60, 80, 0.2);
+    box-shadow: 4px 4px 8px -4px rgba(34, 60, 80, 0.2);
```

Comment:
Are you sure that we need this vendor prefixes? create-react-app supports the vendor prefixes based on the configuration.

Reply:
And `rgba(34, 60, 80, 0.2);` to constants or use a ready-made constant `box-shadow`